api/classes/database.js

```diff
@@ -0,0 +1,16 @@
+const config = require('../../knexfile')['development'];
+
+// reset database at each startup
+//try {
```

Comment:
for dev purposes you might want to drop the db and seed/migrate, if you uncomment this, it will do that on npm start